src/main.rs

```diff
@@ -24,5 +28,6 @@ async fn main() {
 	match args {
 		Command::BuildUpgrade(prefs) => build_upgrade(prefs).await,
 		Command::SubmitReferendum(prefs) => submit_referendum(prefs).await,
+		Command::ScaffoldTests(prefs) => scaffold_tests(prefs).await,
```

Comment:
Just a suggestion, can't think of something else. But "scaffold" isn't really a verb.
```suggestion
		Command::GenerateTestScaffold(prefs) => GenerateTestScaffold(prefs).await,
```

Reply:
fixed, thanks